forms-flow-theme/scss/_button.scss

```diff
@@ -22,4 +22,17 @@
 
 .btn-link:hover {
     text-decoration: underline !important;
+}
+
+.btn-primary:focus-visible,
+.btn-primary:focus,
+.btn.btn-link:focus-visible,
+.btn.btn-link:focus,
+.btn-info:focus-visible,
+.btn-info:focus,
+.btn-light:focus-visible,
+.btn-light:focus,
+.btn-outline-primary:focus-visible,
+.btn-outline-primary:focus {
+    outline: 3px solid var(--ff-primary-200) !important;
```

Comment:
can we use the variable directly this page instead of using (--ff-primary-200), ?

Reply:
No. $primary-200 is a custom variant that we made on the $primary. at the same time, $gray-200 will work as it is a bootstrap variable. that's why  I took it from the root variable. (had tried the other way as well).